### DISCUSSION TITLE
Testing a local Hugging Face model - Configuration File Required?

### DISCUSSION THREAD

**Discussion by siftxxx** [open-compass/opencompass#1075] [Q&A]
I would like to test a local Hugging Face model. Do I need to create a configuration file for this purpose? If so, is there an existing config file that I can reuse or reference?

Any guidance on setting up a local model for testing would be greatly appreciated.

Thank you!

**Comment by jingmingzhuo** [comment]
Just an example:
Asumming that you want to test your local hf_llama3_8b_instruct, you can copy the official config, configs/models/hf_llama/hf_llama3_8b_instruct.py, and change the 'path' parameter to your local path.
